helpers/assert_file_is_empty

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+. bootrr
+
+TEST_CASE_ID="$1"
+FILEPATH="$2"
+TIMEOUT="${4:-1}"
+
+if [ -z "${TEST_CASE_ID}" -o -z "${FILEPATH}" ]; then
+	echo "Usage: $0 <test-case-id> <file path> [<timeout>]"
+	exit 1
+fi
+
+timeout ${TIMEOUT} [ -s ${FILEPATH} ] || test_report_exit pass
```

Comment:
probably better with double quotes around ${FILEPATH}? it looks like we should do that in other places..

Reply:
sure I will change